src/ResourceManager/MachineLearningCompute/Commands.MachineLearningCompute.Test/ScenarioTests/MLCTests.ps1

```diff
@@ -302,4 +302,27 @@ function Test-Set
 
     # Cleanup
     TeardownTest -ResourceGroupName $resourceGroupName
-}
\ No newline at end of file
+}
+
+function Test-RemoveIncludeAllResources
+{
+    $resourceGroupName = GetUniqueName("mlcrp-cmdlet-test-remove-all")
+    $clusterName = GetUniqueName("mlcrp-cmdlet-test-remove-all")
+
+    SetupTest $resourceGroupName
+
+    # Create the cluster
+    $cluster = GetDefaultLocalClusterProperties
+    $createdCluster = New-AzureRmMlOpCluster -ResourceGroupName $resourceGroupName -Name $clusterName -Cluster $cluster
+
+	# Get the managed by resource group name before deleting
+	$managedByResourceGroupName = GetManagedByResourceGroupName -ResourceGroupName $resourceGroupName
+
+	# Delete the cluster
+	Remove-AzureRmMlOpCluster -ResourceGroupName $resourceGroupName -Name $clusterName -IncludeAllResources
+
+    Assert-Throws ( Get-AzureRmResourceGroup -ResourceGroupName $managedByResourceGroupName )
```

Comment:
Why does this throw? Shouldn't it complete the operation successfully and delete both the managed resource group and the cluster? And if it is supposed to throw, I would think you'd want another test with the command running successfully.

Reply:
This throws because after a successful remove call with -IncludeAllResources the resource group I'm trying to get should not exist. So by expecting the throw I can verify that the -IncludeAllResources worked correctly.

Reply:
Ah, that makes sense.  This is fine then.